### ISSUE TITLE
How about Turtle

### ISSUE THREAD

**Issue by VladimirAlexiev** [scienceai/scholarly-article#22]
Subject says it all.
(Is it common to define ontologies in a JSONLD context? I didn't know that)

**Comment by sballesteros** [comment]
Note that the ontology is not really defined in a JSON-LD context. We just use JSON-LD as one of the possible serialization of RDF. For instance, we also provide an RDFa serialization (view source on `http://ns.science.ai`).  I am not sure how common that is. Out of memory i can think of https://www.hydra-cg.com/spec/latest/core/#the-hydra-core-vocabulary-in-json-ld as another example. 
We would be happy to provide additional serialization if helpful!

**Comment by VladimirAlexiev** [comment]
Yes, publish RDF/XML (mandatory) and Turtle (highly recommended), and serve them with content negotiation.
http://rdf-translator.appspot.com does all sort of conversions, and you can automate easily, eg
```sh
http://rdf-translator.appspot.com/convert/rdfa/n3/http://ns.science.ai > scholarly-article.n3
```
n3 is almost the same as Turtle except `=` should be `owl:sameAs`.

Buggies:
- `sa:Acknowledgements = <http://purl.org/spar/deo/Acknowledgements> .`
Replace with `owl:equivalentClass` as `=` (`owl:sameAs`) is intended for individuals
- http://purl.org/spar/doco/ etc:
Define and use prefixes for all referenced ontologies. (eg http://prefix.cc/doco says that's `doco:`)
- `@prefix sa: <http://ns.science.ai#> .`
Since your ontology is served by http://ns.science.ai/ (if you agree to fix #24), so the prefix should include slash: `http://ns.science.ai/#`.
It's a bit unusual to have both slash and hash, but that's ok; unless you're willing to change to eg `http://ns.science.ai/ontology#`
- `rdfs:isDefinedBy "https://ns.science.ai/"@en-us ;`
Instead of literal, you should use a URL. The best practice is to use precisely the ontology prefix:
`rdfs:isDefinedBy sa: ;`
- `dc:source <https://github.com/scienceai/scholarly-article/issues/20> ;`
Change to `dct:source`, which is an object property (dc: is merely rdf:Property)
- include `rdfs:label` for each term.
In particular, what's the point of providing altLabel if you don't have label.
Note: unfortunately both skos:prefLabel and skos:altLabel are subprops of rdfs:label, so you may want to use skos:prefLabel for your main labels
- comments and labels use `@en-us`. Use simply `@en` because I don't see any US English specific constructs.
